README.md

```diff
@@ -1,74 +1,23 @@
-![ddev logo](images/ddev_logo.png)
-
+# ddev
 [![CircleCI](https://circleci.com/gh/drud/ddev.svg?style=shield)](https://circleci.com/gh/drud/ddev) [![Go Report Card](https://goreportcard.com/badge/github.com/drud/ddev)](https://goreportcard.com/report/github.com/drud/ddev) ![project is maintained](https://img.shields.io/maintenance/yes/2017.svg)
 
-# ddev
+![ddev logo](images/ddev_logo.png)
 
 ddev is an open source tool that makes it dead simple to get local PHP development environments up and running within minutes. It's powerful and flexible as a result of its per-project environment configurations, which can be extended, version controlled, and shared. In short, ddev aims to allow development teams to use Docker in their workflow without the complexities of bespoke configuration.
 
-## Roadmap
-
-Each DRUD product has a dedicated product owner, who serves as the primary advocate for customers and end-users when making decisions regarding the public roadmap. For the [ddev roadmap](https://github.com/drud/ddev/wiki/roadmap), @rickmanelius is currently serving as the product owner.
-
-We use the longer-term roadmap to prioritize short-term sprints. Please review the [ddev roadmap](https://github.com/drud/ddev/wiki/roadmap) and [ddev issue queue](https://github.com/drud/ddev/issues) to see what's on the horizon.
+## Getting Started
 
-## System Requirements
+1. **Check System Requirements:** We support macOS High Sierra, Windows 10, and Linux ([more info here](https://ddev.readthedocs.io/en/latest/#system-requirements)).
```

Comment:
Actually, we support Sierra and higher. In reality we support any place docker-for-windows, docker-for-mac, or Linux docker will run. We *test* on High Sierra. The only change we actually need to make is to say Sierra and higher, or maybe El Capitan and higher.

Reply:
I'm happy to make this more exact by calling out Sierra and above and most Linux distributions that support docker.